perses/tests/test_hybrid_builder.py

```diff
@@ -465,148 +217,6 @@ def run_endpoint_perturbation(lambda_thermodynamic_state, nonalchemical_thermody
 
     return [df, ddf_corrected, Neff_max]
 
-def get_available_parameters(system, prefix='lambda'):
-    parameters = list()
-    for force_index in range(system.getNumForces()):
-        force = system.getForce(force_index)
-        if hasattr(force, 'getNumGlobalParameters'):
-            for parameter_index in range(force.getNumGlobalParameters()):
-                parameter_name = force.getGlobalParameterName(parameter_index)
-                if parameter_name[0:(len(prefix)+1)] == (prefix + '_'):
-                    parameters.append(parameter_name)
-    return parameters
-
-def build_two_residues():
```

Comment:
It's a lot of clutter, and I don't really know what's going on in half of it. I think for the protein mutation stuff it will be quicker to just start again. If not, it's in the commit history and can be resurrected from there.